src/version.ts

```diff
@@ -1,4 +1,4 @@
-export function version () {
+export function version (): void {
   // @ts-expect-error: __build__ is injected at build time
```

Comment:
Instead of `@ts-expect-error`, can we please define `__build__` in a definition file?

Reply:
Tried all options to get TypeScript to recognize `__build__` – including updating `tsconfig.json` with correct `include` and `typeRoots` paths – but still getting `TS2304: Cannot find name '__build__'` on `deno task test`.

Switching to `import './types/build.d.ts'` as a reliable workaround instead of using `// @ts-expect-error`.